### DISCUSSION TITLE
Wasn't found

### DISCUSSION THREAD

**Discussion by Noursultan** [webdriverio/webdriverio#7075] [Q&A]
![image](https://user-images.githubusercontent.com/86630166/123767596-6a758b80-d8e9-11eb-92c5-5be37c90bfea.png)
Hi everyone! Could you explain please why "wasn't found"? I specified right locator.

**Comment by christian-bromann** [comment]
@Noursultan specifying the right locator doesn't mean that the element is there when you query it. In your case you missed an `await` in line 67 of your screenshot, so WebdriverIO queried it before the value was set.

**Comment by cg-tester** [comment]
@christian-bromann 
if I remove code  services:['chromedriver'] in wdio.conf.js file,  browser.maximizeWindow() function does not work.  Could you explain why ?

**Reply by christian-bromann** [reply]
Make sure you are using the latest version of WebdriverIO. Driver management support was added in recent version.
